spec/client/FilterButtons.spec.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+import fetchMock from 'fetch-mock';
+
+import dummyData from '../../client/parklets.json';
+import App from '../../client/src/App';
+
+const url = 'https://data.sfgov.org/resource/6a7x-cttf.json';
+
+describe('Filter Buttons', () => {
+  fetchMock.getOnce(url, {
+    status: 200,
+    body: dummyData,
+  });
+
+  const wrapper = shallow(<App />);
+
+  test('All parklets should be loaded with no filters', () => {
+    const { activeParklets } = wrapper.state();
+    expect(activeParklets).toHaveLength(32);
+  });
+
+  test('Only parklets with wifi active with only wifi filter on', () => {
+    wrapper.instance().handleFilters('wifiFilter');
+    const { activeParklets, hiddenParklets } = wrapper.state();
+    const wifiOnly = [...activeParklets, ...hiddenParklets].filter(parklet => parklet.wifi);
+
+    expect(activeParklets).toHaveLength(wifiOnly.length);
```

Comment:
I actually forgot about that portion of the test, at the time of writing it I did not want to count nor rationalize how many parklets are there in an active filter button, so I used a filter function to filter the same amount of parklet as it does during the state since I'm also working on multiple active filter button as well. I should probably change it to hardcode in that case.  What are your thoughts on this? (I actually might change it to hardcode if I have time before I sleep)